falcon/response.py

```diff
@@ -123,6 +133,103 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def set_cookie(self, key, value, **cookie_params):
+        """Set a response cookie.
+
+        Args:
+            key (str): Cookie key name
```

Comment:
http://tools.ietf.org/html/rfc6265 refers to it as "cookie name" so i agree that would be the right naming.